hw1/02-palindrome.js

```diff
@@ -1,3 +1,33 @@
-const elem = document.querySelector('input');
+function isPalindrome(value) {
+  const stringInput = value.toString();
+  const reversedInput = stringInput.split("").reverse().join("");
+  return stringInput === reversedInput;
+}
 
-elem.addEventListener('input', handleInput);
+const displayPalindromeResult = (resultColorClass, resultTextString) => {
+  const displayResult = document.getElementById("result");
+  displayResult.className = resultColorClass;
+  displayResult.textContent = resultTextString;
+};
+const elem = document.querySelector("input");
+
+const handleInput = () => {
+  const input = parseInt(elem.value);
+
+  if (!Number.isNaN(input)) {
```

Comment:
Rule [19.9](https://github.com/airbnb/javascript#whitespace--padded-blocks) states that blocks should not be padded with whitespace